### DISCUSSION TITLE
Variants without any channel reference

### DISCUSSION THREAD

**Discussion by Anubiella** [saleor/saleor#9731] [Deployment]
Hi everyone,

I've faced a strange behaviour dealing with product variants.
After an usual size variant mass creation by the variant wizard into the product page(with channel prices and stocks, see images),
![DMT KR0 ROAD CYCLING SHOES - Ale Bike Wear 10-5-2022 12-30-07](https://user-images.githubusercontent.com/31857092/167608801-d9508581-f382-41ae-81f4-838ab4938d1f.png)

![48 - Ale Bike Wear 10-5-2022 12-32-03](https://user-images.githubusercontent.com/31857092/167609660-df6fef9c-92b4-4e93-8ccf-280d14f3eae6.png)

![48 - Ale Bike Wear 10-5-2022 12-32-15](https://user-images.githubusercontent.com/31857092/167609960-56bce6ce-d9b9-4382-b4fe-df7eff925f09.png)

![48 - Ale Bike Wear 10-5-2022 12-32-29](https://user-images.githubusercontent.com/31857092/167610257-28dd366d-59d6-41d7-8536-501c31fd17ad.png)

we've found the necessity to create another variant, **at a later time**.

What appeared is when you create a variant **after all the others** (created by the wizard) you **have no way to deal/set channels and so stocks and prices (see image)**.

![36 - Ale Bike Wear 10-5-2022 12-40-02](https://user-images.githubusercontent.com/31857092/167610945-999843e0-f74b-426c-b5bd-8a060fb93051.png)

The only way you can manage this variant is by calling via API the mutation productVariantChannelListingUpdate , 
only then the variant will be hooked to all che channels and had set the prices and stocks.

Is this the current/normal behavour?
It seems weird to me that the act of creating a new variant after the first ones denies the chance to deal with channels,

**Comment by timuric** [comment]
@Anubiella sorry for the confusing UI, we are in process of redesigning it. You have to go back to the product page and enable your newly created variant 
<img width="1557" alt="image" src="https://user-images.githubusercontent.com/4006792/167803830-15733af9-865d-474d-bafc-c69e2554e2e0.png">

**Comment by Anubiella** [comment]
Thanks a lot, god bless have a nice day!

Kind regards,
Lorenzo
